assets/src/design-system/components/contextMenu/stories/index.js

```diff
@@ -0,0 +1,141 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { action } from '@storybook/addon-actions';
+import { boolean } from '@storybook/addon-knobs';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import ContextMenu from '..';
+import { DarkThemeProvider } from '../../../storybookUtils';
+
+const items = [
+  { label: 'Copy', shortcut: '⌘ X' },
+  { label: 'Paste', shortcut: '⌘ C' },
+  { label: 'Delete', shortcut: 'DEL' },
+  { label: 'Send to back', shortcut: '⌥ ⌘ [', separator: 'top' },
+  { label: 'Send backward', shortcut: '⌘ [', disabled: true },
+  { label: 'Bring forward', shortcut: '⌘ ]', disabled: true },
+  {
+    label: 'Bring to front',
+    shortcut: '⌥ ⌘ [',
+    disabled: true,
+    separator: 'bottom',
+  },
+  { label: 'Copy style', shortcut: '⌥ ⌘ C' },
+  { label: 'Paste Style', shortcut: '⌥ ⌘ V' },
+  { label: 'Clear text styles' },
+  { label: 'Add style to "Saved styles"' },
+  { label: 'Add color to "Saved colors' },
+];
+
+const randomItems = [
+  { label: 'one' },
+  { label: 'two', shortcut: '%C' },
+  { label: 'i am a button!', shortcut: '$$$' },
+  { label: 'neither a button nor a link' },
+  { label: 'this is disabled', disabled: true },
+  { label: 'three', separator: 'top' },
+  {
+    label: 'i am a link!',
+    href: 'https://www.google.com/',
+    shortcut: '⌥ ⌘ A',
+  },
+  {
+    label: 'i am a very very very very very very very long label',
+    separator: 'bottom',
+  },
+  { label: 'lions' },
+  { label: 'tigers', separator: 'top' },
+  { label: 'bears' },
+];
+
+export default {
+  title: 'DesignSystem/Components/ContextMenu',
+  component: ContextMenu,
+};
+
+const Container = styled.div`
+  position: relative;
+  height: 700px;
+  width: 500px;
+  padding: 20px;
+  background-color: ${({ theme }) => theme.colors.bg.primary};
+`;
+
+export const _default = () => {
+  const itemsWithEventHandlers = items.map((item) => ({
+    ...item,
+    onClick: action(`Clicked on \`${item.label}\``),
+  }));
+
+  return (
+    <Container>
+      <ContextMenu
+        items={itemsWithEventHandlers}
+        isOpen={boolean('isOpen', true)}
+        isLTR={boolean('Left To Right', false)}
+      />
+    </Container>
+  );
+};
+
+export const DarkMode = () => {
+  const itemsWithEventHandlers = items.map((item) => ({
+    ...item,
+    onClick: action(`Clicked on \`${item.label}\``),
+  }));
+
+  return (
+    <DarkThemeProvider>
+      <Container>
+        <ContextMenu
+          items={itemsWithEventHandlers}
+          isOpen={boolean('isOpen', true)}
+          isLTR={boolean('Left To Right', false)}
+        />
+      </Container>
+    </DarkThemeProvider>
+  );
+};
```

Comment:
You might just be able to do this to see how much custom rtl support you actually need to add. It does a surprising amount automagically

```suggestion
import { StyleSheetManager, ThemeProvider } from 'styled-components';
import stylisRTLPlugin from 'stylis-plugin-rtl';

export const DarkMode = () => {
  const itemsWithEventHandlers = items.map((item) => ({
    ...item,
    onClick: action(`Clicked on \`${item.label}\``),
  }));

  return (
    <DarkThemeProvider>
      <StyleSheetManager stylisPlugins={boolean('Right to Left', false) ? [stylisRTLPlugin] : []}>
        <Container>
          <ContextMenu
            items={itemsWithEventHandlers}
            isOpen={boolean('isOpen', true)}
          />
        </Container>
      </StyleSheetManager>
    </DarkThemeProvider>
  );
};
```

Reply:
This works great in the actual app (when I removed the `isLTR` styling. 

I think storybook has some challenges when I do it there:

|Storybook|Dashboard|
|--|--|
|<image src="https://user-images.githubusercontent.com/22185279/107705673-556ec780-6c7c-11eb-9b90-d88978dfd304.png" width="200px" />|<image src="https://user-images.githubusercontent.com/22185279/107705625-41c36100-6c7c-11eb-80e1-8ece6414dd48.png" width="200px" />|

Reply:
That's because the app has a "global" `StyleSheetManager` wrapper around it, which we don't have in the editor.

We could look into adding that for all storybook stories as well. For the short term this suggestion seems to work.